tests/integration-tests/arweave-and-3box/test/test.js

```diff
@@ -0,0 +1,80 @@
+const path = require("path");
+const execSync = require("child_process").execSync;
+const { system, patching } = require("gluegun");
+const { createApolloFetch } = require("apollo-fetch");
+
+const Contract = artifacts.require("./Contract.sol");
+
+const srcDir = path.join(__dirname, "..");
+
+const fetchSubgraphs = createApolloFetch({
+  uri: "http://localhost:18030/graphql"
+});
+const fetchSubgraph = createApolloFetch({
+  uri:
+    "http://localhost:18000/subgraphs/name/test/overloaded-contract-functions"
```

Comment:
Might want to change the subgraph name too. Perhaps all tests should have the same `package.json` and subgraph name, just to make copying them easier. 😁

Reply:
Ha that's a good idea.

Reply:
But we may still want to give the deployed subgraphs different names in case we at some point want to run these tests on the same graph-node for some reason. So I'll give it a distinct name.

Reply:
Sounds good